python/pyarrow/tests/util.py

```diff
@@ -456,4 +456,5 @@ def windows_has_tzdata():
     this configurable at run-time)
     """
     tzdata_path = os.path.expandvars(r"%USERPROFILE%\Downloads\tzdata")
-    return os.path.exists(tzdata_path)
+    tzdata_set_path = os.environ.get('PYARROW_TZDATA_PATH', '')
+    return os.path.exists(tzdata_path) or os.path.exists(tzdata_set_path)
```

Comment:
Should we here only test `tzdata_set_path` _if set_ (so that you set the env variable, we don't fallback to check the default location)?

Reply:
https://github.com/apache/arrow/pull/37436/commits/c9fd8612739745373bdadce7a1520982894a9903#diff-2c3ffa32d2769f65282dba5b78ba82e53daa88d8a762079947f242f132e6aa26

Hope I understood correctly.